bitnami/wavefront/README.md

```diff
@@ -0,0 +1,249 @@
+| `proxy.tracePort`                   | Port for distributed tracing data (usually 30000)                                      | `nil`                          |
+| `proxy.jaegerPort`                  | Port for Jaeger format distributed tracing data (usually 30001)                        | `nil`                          |
+| `proxy.traceJaegerHttpListenerPort` | Port for Jaeger Thrift format data (usually 30080)                                     | `nil`                          |
+| `proxy.zipkinPort`                  | Port for Zipkin format distribued tracing data (usually 9411)                          | `nil`                          |
+| `proxy.traceSamplingRate`           | Distributed tracing data sampling rate (0 to 1)                                        | `nil`                          |
+| `proxy.traceSamplingDuration`       | When set to greater than 0, spans that exceed this duration will force trace to be sampled (ms) | `nil`                 |
+| `proxy.histogramPort`               | Port for histogram distribution format data (usually 40000)                            | `nil`                          |
+| `proxy.histogramMinutePort`         | Port to accumulate 1-minute based histograms on Wavefront data format (usually 40001)  | `nil`                          |
+| `proxy.histogramHourPort`           | Port to accumulate 1-hour based histograms on Wavefront data format (usually 40002)    | `nil`                          |
+| `proxy.histogramDayPort`            | Port to accumulate 1-day based histograms on Wavefront data format (usually 40003)     | `nil`                          |
+| `proxy.deltaCounterPort`            | Port to accumulate 1-minute delta counters on Wavefront data format (usually 50000)    | `nil`                          |
+| `proxy.args`                        | Additional Wavefront proxy properties to be passed as command line arguments in the `--<property_name> <value>` format | `nil` |
+| `proxy.heap`                        | Wavefront proxy Java heap maximum usage (java -Xmx command line option)                | `nil`                          |
+| `proxy.preprocessor.rules.yaml`     | YAML configuraiton for Wavefront proxy preprocessor rules                              | `nil`                          |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```bash
+$ helm install my-release \
+  --set proxy.replicas=3 \
+    bitnami/wavefront
+```
+
+The above command sets 3 proxy replicas.
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```bash
+$ helm install my-release -f values.yaml bitnami/wavefront
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+
+## Configuration and installation details
```

Comment:
I would include a section explaining the differences between deploying the Collector as a deployment and a daemonset.